BuildTools/Emscripten/pack-artifacts.py

```diff
@@ -29,6 +29,8 @@
     "GLTFViewer"
 ]
 
+EXCLUDE_ITEMS = ["CMakeFiles", "cmake_install.cmake"]
```

Comment:
Do you also need to include `.o` files here?

Reply:
These files are located at  `CMakeFiles`